api/src/main/java/jakarta/json/bind/JsonbConfig.java

```diff
@@ -126,6 +127,11 @@
      */
     public static final String DESERIALIZERS = "jsonb.derializers";
 
+    /**
+     * Property used to specify custom deserializers.
+     */
```

Comment:
if we drop the two entry points from this class we can make this whole PR an appendix - same as proposing the introspector/annotation based solution - of the spec while there is not agreement on the final outcome.
Can enable to have API proposals and decide later (bean validation went that way in its time - for method validation).
Overall blocking point is to have something unified versus something duplicated for 3rd party libraries and I don't see us choosing before next release since both parties seems to either priviledge the interoperability and simplicity or the application API first (being said the introspector API enables to join the API point later if needed whereas decorator does not and keeps two concurrent models in the spec forever).

Wdyt? (trying to not block but not pollute the API too early without enabling 3rd party cases)

Reply:
What exactly do you mean by "drop the two entry points from this class". I am sorry, but I don't understand what you are trying to say. Please explain.

Reply:
Not make it explicit in the API (ie use yasson.decorators property without a constant nor a wither.